direnvrc

```diff
@@ -119,8 +119,9 @@ use_flake() {
 }
 
 use_nix() {
-  local path layout_dir
+  local path layout_dir grep
   path=$("${NIX_BIN_PREFIX}nix-instantiate" --find-file nixpkgs)
+  grep=$("${NIX_BIN_PREFIX}nix-build" -E 'with import <nixpkgs> {}; gnugrep')/bin/grep
```

Comment:
I am bit worried about people not having NIX_PATH set with nixpkgs
What do you think about this solution? https://github.com/nix-community/nix-direnv/pull/64

Reply:
Tested #64 and it worked for me on OSX (didn't test on linux, but it'll be fine I'm sure) so I'll close this. Thanks!